packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -108,3 +82,38 @@ describe("SharedTree", () => {
         });
     });
 });
+
+/**
+ * Inserts a single node under the root of the tree with the given value.
+ * Use {@link getTestValue} to read the value.
+ */
+function insertTestValue(tree: ISharedTree, value: TreeValue): void {
+    // Apply an edit to the first tree which inserts a node with a value
+    tree.runTransaction((forest, editor) => {
+        const writeCursor = singleTextCursor({ type: brand("TestValue"), value });
+        editor.insert({
+            parent: undefined,
+            parentField: detachedFieldAsKey(forest.rootField),
+            parentIndex: 0,
+        }, writeCursor);
+
+        return TransactionResult.Apply;
+    });
+}
+
+/**
+ * Reads a value in a tree set by {@link insertTestValue} if it exists
+ */
+function getTestValue(tree: ISharedTree): TreeValue | undefined {
+    const readCursor = tree.forest.allocateCursor();
+    const destination = tree.forest.root(tree.forest.rootField);
+    const cursorResult = tree.forest.tryMoveCursorTo(destination, readCursor);
+    const { value } = readCursor;
+    readCursor.free();
+    tree.forest.forgetAnchor(destination);
+    if (cursorResult === TreeNavigationResult.Ok) {
+        return value;
+    }
```

Comment:
This function doesn't include the check against unexpected elements that was present before.
`assert.equal(readCursor.seek(1), TreeNavigationResult.NotFound);`
I think Craig added that check after catching a bug.
We could either add the check here (which would make the function unusable for cases where we expect more nodes to be present) or have the function return an array of the values at the root. I don't have a strong opinion but I do think we want to keep the coverage.

Reply:
Ah, okay. I was wondering about that. If it was to catch a specific bug then we should make a specific regression test for it. I'll add it back in another test.